packages/ie11-custom-properties/LICENSE

```diff
@@ -0,0 +1,15 @@
+@fluentui/ie11-custom-properties
+
+Copyright (c) Microsoft Corporation
```

Comment:
@dzearing What should we do about the copyright and authorship notice since this is forked from another package?